src/crepe/manager/LoopTimerManager.h

```diff
@@ -45,19 +49,50 @@ class LoopTimer {
 	int get_fps() const;
 
 	/**
-	 * \brief Get the current game scale.
+	 * \brief Get the current time scale.
 	 *
-	 * \return The current game scale, where 0 = paused, 1 = normal speed, and values > 1 speed
+	 * \return The current time scale, where (0 = pause, < 1 = slow down, 1 = normal speed, > 1 = speed up).
 	 * up the game.
 	 */
-	double get_game_scale() const;
+	double get_time_scale() const;
+
+	/**
+	 * \brief Set the time scale.
+	 *
+	 * time_scale is a value that changes the delta time that can be retrieved using get_delta_time function. 
+	 * 
+	 * \param time_scale The desired time scale (0 = pause, < 1 = slow down, 1 = normal speed, > 1 = speed up).
+	 */
+	void set_time_scale(double time_scale);
+
+	/**
+	 * \brief Get the scaled fixed delta time om seconds.
+	 *
+	 * The fixed delta time is used for operations that require uniform time steps, 
+	 * such as physics calculations, and is scaled by the current time scale.
+	 *
+	 * \return The fixed delta time, scaled by time scale, in seconds.
+	 */
+	double get_fixed_loop_interval() const;
```

Comment:
Misschien kan de naamgeving hier (en op andere plekken in deze klasse) iets duidelijker en vooral consistenter. Gebruik bijvoorbeeld overal de prefix `get_`/`set_` met daarna `fixed_loop`/`frame_update` en dan `delta_time`/`scaled_delta_time` (o.i.d.).

Reply:
ik snap niet helemaal wat je bedoeld (ik zag dat ik de doxygen wel had omgewisseld tussen twee functies dus dat heb ik al wel gefixt). Bij iedere functie die gebruikt kan worden door de user/systems staat get_ en set_

Reply:
Ja klopt, de `get_` en `set_` prefix gebruik je goed. Maar ik doel op hetgeen wat daarna volgt (dat is naar mijn idee niet erg consistent).
Je hebt bijvoorbeeld de methode `get_delta_time()` waarbij niet direct duidelijk wordt of het gaat om de fixed_loop of frame_update.
Je gebruikt één keer `interval` terwijl je hier volgens mij ook delta_time bedoeld.
Zou je niet overal dezelfde soort benaming kunnen gebruiken zodat de functienamen wat duidelijker en eenduidiger worden?